### PR TITLE
feat: add event sync correctness tests

### PR DESCRIPTION
**Pull request by nathanielc**
This change adds correctness tests that hit the Ceramic API directly for events. Events are written and validated to have propagated to other nodes.

TODO:
- [x] Wait for streamid to release
- [x] Wait for https://github.com/ceramicnetwork/rust-ceramic/pull/238 to merge as it contains fixes that make the tests pass

### CODE REVIEW

File: suite/src/__tests__/fast/events.test.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    let secondHalf = modelEvents.slice(half, modelEvents.length)
+    await writeEvents(firstNodeUrl, firstHalf)
+
+    // Now subscribe on the other nodes
+    for (let idx = 1; idx < CeramicUrls.length; idx++) {
+      let url = CeramicUrls[idx]
+      await subscribe(url, modelID)
+    }
+    // Write the second half of the data
+    await writeEvents(firstNodeUrl, secondHalf)
```
Comment (smrz2001) [inline suite/src/__tests__/fast/events.test.ts:126]: The behavior should be symmetric so it shouldn't matter but do we want to write the second half to a different node besides the first one? Just a thought.

Comment (nathanielc) [reply suite/src/__tests__/fast/events.test.ts:126]: I like that, I think I'll add an active writes on two nodes test

**Review by smrz2001 — APPROVED** [review]
🥳 🚀

**Review by smrz2001 — APPROVED** [review]
:shipit:

**Pull request merged**